### PR TITLE
Vn 165 create table that maps our entities to salesforce entities with external ids

### PR DESCRIPTION
**Pull request by SegevHaviv**

### CODE REVIEW

File: src/graphql/whitelist.graphql [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# import Query.dummyEntity, Query.dummyEntities, Query.dummyEntitiesConnection, Mutation.createDummyEntity, Mutation.updateDummyEntity, Mutation.deleteDummyEntity from './generated/openCrudSchema.graphql'
+# import Query.vennMacforceExternalId, Query.vennMacforceExternalIds, Query.vennMacforceExternalIdsConnection, Mutation.createVennMacforceExternalId, Mutation.updateVennMacforceExternalId, Mutation.deleteVennMacforceExternalId from './generated/openCrudSchema.graphql'
```
Comment (VinoLora) [inline src/graphql/whitelist.graphql:1]: Usually it's one line ,  is this ok?

File: src/graphql/datamodel.graphql [hunk 2]
```diff
@@ -14,12 +14,23 @@ enum OPERATION {
...
+  deleted:                Int!                    @default(value: 0)
+}
+
+type VennToMacforceMapping @pgTable(name: "venn_to_macforce_mappings") {
   id:                     ID!                     @unique
-  name:         String          @pgColumn(name: "name")
-  # TODO: the entity fields go here
-  # myScalarField         SomeType                @pgColumn(name: "my_scalar_field")
-  # myRelationField       SomeOtherEntityType     @pgRelation(column: "my_relation_field_column")
+  dummy:                  String                  @pgColumn(name: "dummy")
```
Comment (VinoLora) [inline src/graphql/datamodel.graphql:31]: Remind me why do we need 'dummy' field?

**Review by VinoLora — APPROVED** [review]

File: src/graphql/whitelist.graphql (hunk 1 above)
Comment (Grishezz) [reply src/graphql/whitelist.graphql:1]: it should be fine, i think we've turned it into a one line as an optimisation in persistency, but in smaller services and maybe generally as we moved it to ECS it's no longer needed and multiline is more readable

File: src/graphql/datamodel.graphql (hunk 2 above)
Comment (Grishezz) [reply src/graphql/datamodel.graphql:31]: i think it's a legacy from prisma times, which we most likely no longer need

Comment (SegevHaviv) [reply src/graphql/datamodel.graphql:31]: To run test-helper tests & to have a string value for DAO tests, vennToMacforceMapping doesn't contain any string values, only joined entities.

Comment (Grishezz) [reply src/graphql/datamodel.graphql:31]: ah, my bad, i missed the context of this comment.

**Pull request merged**